experimental/WreathMacdonaldpols/src/WreathMacdonaldpols.jl

```diff
@@ -0,0 +1,202 @@
+# I would like to credit Dario Mathiä who produced an initial version of the following code
+
+export wreath_macs, fake_deg
+
+Oscar.canonical_unit(x::QQAbFieldElem{AbsSimpleNumFieldElem}) = x #meanwhile canonical_unit gets fixed
+
+# Tools
+
+#gives indices of a subarray
+function subarray_indices(sub,arr)
+  return [findfirst(==(s), arr) for s in sub]
+end
+
+# Computes the b-invariant
+function b1(lambda::Vector{Int})
+  return sum((i-1)*l_i for (i, l_i) in enumerate(lambda); init=0)
+end
+
+# Computes the b-invariant of a multipartition
+function b_inv(lbb::Multipartition)
+  return b1(map(lambda -> sum(lambda)[1], lbb)) + length(lbb)*sum(i -> b1(data(lbb[i])),1:length(lbb))
```

Comment:
```suggestion
  return b1(map(sum, lbb)) + length(lbb)*sum(lambda -> b1(data(lambda)), lbb)
```
some more simplification